libs/components/ag-grid/src/lib/modules/ag-grid/top-horizontal-scroll/top-horizontal-scroll.service.ts

```diff
@@ -0,0 +1,76 @@
+import { ElementRef, Injectable, OnDestroy } from '@angular/core';
+import { MutationObserverService } from '@skyux/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TopHorizontalScrollService implements OnDestroy {
+  private mutationObserver: MutationObserver;
+
+  constructor(private mutationObserverSvc: MutationObserverService) {}
```

Comment:
Use hash-named properties and assign them with the constructor parameter values instead of the private constructor parameter convenience feature of TypeScript.

```
#mutationObserverSvc: MutationObserverService;

constructor(mutationObserverSvc: MutationObserverService) {
  this.#mutationObserverSvc = mutationObserverSvc;
}
```